### ISSUE TITLE
Create Navbar component

### ISSUE THREAD

**Issue by siyamregn777** [siyamregn777/portfolio#10]
A global Navbar should be added to provide navigation between the Home, About, and Contact pages. It should be styled for readability and accessibility and should be included across all pages of the website.
